tests/test_ReactForRole.py

```diff
@@ -756,9 +757,9 @@ async def test_rfr_edit_thumbnail_bad_attach(attach):
                     mock.AsyncMock(return_value=(message, channel))):
         with mock.patch('cogs.ReactForRole.ReactForRole.get_embed_from_message', return_value=embed):
             with mock.patch('cogs.ReactForRole.ReactForRole.prompt_for_input', return_value=attach):
-                with pytest.raises(discord.ext.commands.BadArgument):
+                with pytest.raises(Exception):
```

Comment:
Exception is a bit too vague, worth changing to be more specific

Reply:
46cc61aa4e20443462434d50efcd4f4050140265 handles that, since BadArgument isn't the only exception that can be raised by it, replaced it with a ClientError from aiohttp that includes all possible (I think) errors raised from this.